docs/fleet/index.md

```diff
@@ -0,0 +1,43 @@
+title: Fleet Overview
+description: Overview of Fleet - Sematext Agent management panel
+
+The Fleet section of Sematext Cloud gives you an overview of all the Sematext Agents that are running in your systems.
+![Fleet screen](../images/fleet/fleet.png)
+
+At first glance, you can see the following data in the table:
+* *Host*: Name of the hosts in which the Sematext Agent is installed. Clicking on the name will open the Agent Information Panel.
+* *Version*: Installed version of the Sematext Agent and whether important updates are available. If they are, you can update the agent by just clicking the link and following the instructions. You can also find those instructions [here](https://sematext.com/docs/monitoring/spm-faq/#agent-updating).
+* *Status*: Indicates whether the Sematext Agent is reporting any errors.
+* *Started*: Time elapsed since the Sematext Agent began sending data to Sematext Cloud.
+* *Infra App*: the Infra App which is receiving the data from the Sematext Agent. Clicking on the app name will navigate you to the overview of the corresponding app in the Infrastructure section of Sematext Cloud.
```

Comment:
app => App
Infrastructure could be hyperlinked to https://sematext.com/docs/monitoring/infrastructure/